### PR TITLE
htmlmin: init at 0.1.10

### PR DESCRIPTION
**Pull request by ahmedtd**
###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested ability to import and use package in python interpreter
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -7748,6 +7748,26 @@ in {
     };
   });
 
+
+  htmlmin = buildPythonPackage rec {
```
Comment (Mic92) [inline pkgs/top-level/python-packages.nix]: Please put this expression into a separate file in ./pkgs/development/python-modules

Comment (ahmedtd) [inline pkgs/top-level/python-packages.nix]: Just curious, is there some criterion for which packages go where?  Or are they all being gradually migrated to python-modules?

Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: They all go gradually there.

**Comment by joachifm** [comment]
Thank you

**Pull request merged**